src/editor.c

```diff
@@ -1953,7 +1953,17 @@ gboolean editor_show_calltip(GeanyEditor *editor, gint pos)
 		return FALSE;
 
 	word[0] = '\0';
+
```

Comment:
you need to allow space(s) between the `<>` and the `(`, something like this:
```c
if (pos > 0 && isspace(sci_get_char_at(sci, pos - 1)))
	pos--;
```
(or even a `while` to allow any number of spaces, not only one, but we don't have it currently so probably not necessary)